library/old_protocol/translate_request.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <atomic>
+
+#include "library/n2_protocol.hpp"
+#include "serialize.hpp"
+
+namespace ioremap { namespace elliptics { namespace n2 {
+
+class replier_base {
+public:
+	replier_base(const char *handler_name, dnet_net_state *st, const dnet_cmd &cmd);
+	void reply(std::unique_ptr<n2_message> msg);
+	void reply_error(int errc);
+
+protected:
+	virtual void reply_impl(std::unique_ptr<n2_message> msg);
```

Comment:
why is it protected? Why do inhereted classes would want to call that particular implementation?

Reply:
* It mustn't be public
* It can't be private, otherwise inherited class can't override it

Reply:
upd
Yes, making it `private` is possible, I done it.